cluster-provision/images/vm-image-builder/alpine-cloud-init/create-image.sh

```diff
@@ -12,10 +12,10 @@ if [ ! -f alpine-make-vm-image ]; then
     chmod 755 alpine-make-vm-image
 fi
 
-docker run --platform=$PLATFORM --privileged -v $(pwd):$(pwd):z alpine ash -c "cd $(pwd) &&
+docker run --rm --platform=$PLATFORM -v /lib/modules:/lib/modules -v /dev:/dev --privileged -v $(pwd):$(pwd):z alpine ash -c "cd $(pwd) &&
```

Comment:
nit: you can pass `--workdir $(pwd)` instead of passing it to `sh -c `

Just to understand, why is it necessary to mount /lib/modules and /dev? is it safe? 
would it work if we mount it as read only?

Reply:
--privileged is supose to be mounting those already don't know why I added them, I will remove and use --workdir.